charts/ceph-csi-cephfs/templates/csidriver-crd.yaml

```diff
@@ -8,4 +7,4 @@ metadata:
 spec:
   attachRequired: false
   podInfoOnMount: false
-{{ end }}
+{{- end -}}
```

Comment:
GitHub is showing missing newline at end of file

Reply:
I removed all newlines since they are useless in the templates, the -}} would chomp it anyways

Reply:
this is required to make Github happy, can we revert it back?

Reply:
Will do :)